tests/test_voltage_initializer.py

```diff
@@ -19,7 +19,12 @@ def test_parameters():
     params.add_variable_two_windings_transformers(["twt1", "twt2"])
 
     params.add_algorithm_param({"foo": "bar", "bar": "bar2"})
-    params.add_specific_voltage_limits({"vl_id": (0.5, 1.2)})
```

Comment:
Yes I understand your solution but this is a breaking change. The rule is to never introduce breaking changes for python users. @geofjamg what should we do in that case?

Reply:
maybe we can keep also the previous python method and add a deprecated tag

Reply:
See with @geofjamg, can you also keep the previous method to avoid breaking change?

Reply:
Yes no problem, and I can add "deprecated" on the previous one. I guess the reason for deprecation being to avoid having multiplesimilar API with similar names for same kind of behavior.